src/graphs/SmallBarStackGraph.jsx

```diff
@@ -25,7 +25,8 @@ const handleMouseOver = (event, datum, showTooltip) => {
 export default withTooltip(
```

Comment:
Please add documentation for "showTooltip" in "handleMouseOver" function (line 16). Can't add comment there